src/SFA.DAS.EmployerIncentives.Web/Services/Security/AccountEncodingService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using SFA.DAS.Encoding;
+
+namespace SFA.DAS.EmployerIncentives.Web.Services.Security
+{
+    public class AccountEncodingService : IAccountEncodingService
+    {
+        private readonly IEncodingService _encodingService;
+        public AccountEncodingService(IEncodingService encodingService)
+        {
+            _encodingService = encodingService;
+        }
+
+        public string Encode(long value)
+        {
+            return _encodingService.Encode(value, EncodingType.AccountId);
```

Comment:
Should this be EncodingType.PublicAccountId ?

And be EncodingType.PublicAccountLegalEntityId  when the AccountLegalEntityId   is being encoded/decoded ?